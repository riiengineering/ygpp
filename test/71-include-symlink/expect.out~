This test tests nested includes with symbolic links.

This is included file 1.

This is sub1/file1.in.
This is sub1/file2.in.

This is complex1/file1.in.
This is complex1/file2.in.
This is complex1/file3.in.
This is complex1/file4.in.
