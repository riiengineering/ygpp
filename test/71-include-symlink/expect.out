This test tests nested includes with symbolic links.

This is included file 1.

This is sub1/file1.in.
This is sub1/file2.in.

This is complex1/file1.in.
This is complex1/file2.in.
This is complex1/file3.in.
This is complex1/file4 with\nline break.in.
This is complex1/file5.in\r\r.
This is complex1/file6.in\n\n.
